test/integration/scenarios/07.helloWorld/magefile.go

```diff
@@ -19,19 +19,26 @@ import (
 
 // Test config
 const (
-	NAMESPACE = "sample-k8s-scope"
-	TEST_NAME = "Symphony Hello World sample test scenario"
+	TEST_NAME = "Symphony sample test scenario"
```

Comment:
change the folder name to maestro test since it no longer holds the hello world sample